cmd/zfs/zfs_iter.c

```diff
@@ -298,10 +305,15 @@ zfs_sort(const void *larg, const void *rarg, void *data)
 	zfs_sort_column_t *sc = (zfs_sort_column_t *)data;
 	zfs_sort_column_t *psc;
 
+	if (sc && sc->sc_fast_txg && l && r) {
+		return ((zfs_get_createtxg(l) < zfs_get_createtxg(r)) ?
```

Comment:
This can use `zfs_prop_get_int` to avoid changing the libzfs ABI.

Reply:
Updated!